OrbitLinuxTracing/Tracer.cpp

```diff
@@ -23,13 +23,14 @@ Tracer::Tracer(pid_t pid, double sampling_frequency,
 void Tracer::Run(pid_t pid, uint64_t sampling_period_ns,
                  const std::vector<Function>& instrumented_functions,
                  TracerListener* listener, bool trace_context_switches,
-                 bool trace_callstacks, bool trace_instrumented_functions,
-                 bool trace_gpu_driver,
+                 bool trace_callstacks, bool trace_fp_callstacks,
```

Comment:
+1 to using an enum for the method of callstack unwinding. Overall, I'd probably use a boolean for "trace_callstacks" and have an additional enum 

enum UnwindingMethod {
  kUndefined = 0, 
  kFramePointers,
  kDwarf 
};

or something similar.

At this point it may also make sense to introduce a struct for bundling all tracing options together: 

struct TracingOptions {
  [...]
};

Reply:
Added both, the enum and the struct. Thanks for the suggestion.